### PR TITLE
Chat phase 2

### PR DESCRIPTION
**Pull request by PeterAronov**
phase 2 of the project

### CODE REVIEW

File: backend/api/v1/messages.controller.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    } else if (message.length > 1) {
+        return res.status(status.INTERNAL_SERVER_ERROR).send()
+    } else {
+        return res.status(status.OK).json({
+            message: message[0]
+        });
+    }
+};
+
+const postNewMessage = (req, res) => { //http://localhost:8000/messages/
```
Comment (matan-co) [inline backend/api/v1/messages.controller.js]: name the function by its real puppose, createNewMessage.

File: backend/api/v1/messages.controller.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+    messagesFileService.writeMessagesToJsonFile(res, messages, messagesJsonPath);
+};
+
+const updateMessage = (req, res) => { //http://localhost:8000/messages/88ab2e95-1955-482e-9107-bf2ae8825baf
+    const { id } = req.params;
+    const { text: newText } = req.body;
+    const index = messages.findIndex((message) => message.id === id);
+
+    if (index !== -1) {
```
Comment (matan-co) [inline backend/api/v1/messages.controller.js]: index !== -1 this line is not readable. use other function rather then findIndex

**Review by matan-co — COMMENTED** [review]
I started to go over. will continue later. good job in general

File: backend/api/v1/messages.controller.js (hunk 1 above)
Comment (PeterAronov) [reply backend/api/v1/messages.controller.js]: Done!

File: backend/api/v1/messages.controller.js (hunk 2 above)
Comment (PeterAronov) [reply backend/api/v1/messages.controller.js]: Used find method instead and then deleted/updated by the index

**Review by matan-co — APPROVED** [review]

**Comment by PeterAronov** [comment]
Merged!

**Pull request merged**